src/cf-forms/usage.md

```diff
@@ -22,39 +22,44 @@ Capital Framework.
+        - [Text input disabled state](#text-input-disabled-state)
         - [Full-width modifier](#full-width-modifier)
     - [Basic checkboxes](#basic-checkboxes)
-        - [Default state](#default-state)
-        - [Hover/focus state](#hoverfocus-state)
-        - [Selected state](#selected-state)
-        - [Disabled state](#disabled-state)
+        - [Checkbox default state](#checkbox-default-state)
+        - [Checkbox hover state](#checkbox-hover-state)
+        - [Checkbox focus state](#checkbox-focus-state)
+        - [Checkbox selected state](#checkbox-selected-state)
+        - [Checkbox disabled state](#checkbox-disabled-state)
     - [Basic radio buttons](#basic-radio-buttons)
-        - [Default state](#default-state)
-        - [Hover/focus state](#hoverfocus-state)
-        - [Selected state](#selected-state)
-        - [Disabled state](#disabled-state)
+        - [Radio button default state](#radio-button-default-state)
+        - [Radio button hover state](#radio-button-hover-state)
+        - [Radio button focus state](#radio-button-focus-state)
+        - [Radio button selected state](#radio-button-selected-state)
+        - [Radio button disabled state](#radio-button-disabled-state)
     - [Large target area checkboxes](#large-target-area-checkboxes)
-        - [Default state](#default-state)
-        - [Hover/focus state](#hoverfocus-state)
-        - [Selected state](#selected-state)
-        - [Disabled state](#disabled-state)
+        - [Large checkbox default state](#large-checkbox-default-state)
+        - [Large checkbox hover state](#large-checkbox-hover-state)
+        - [Large checkbox focus state](#large-checkbox-focus-state)
+        - [Large checkbox selected state](#large-checkbox-selected-state)
+        - [Large checkbox disabled state](#large-checkbox-disabled-state)
     - [Large target area radio buttons](#large-target-area-radio-buttons)
-        - [Default state](#default-state)
-        - [Hover/focus state](#hoverfocus-state)
-        - [Selected state](#selected-state)
-        - [Disabled state](#disabled-state)
+        - [Large radio button default state](#large-radio-button-default-state)
+        - [Large radio button hover state](#large-radio-button-hover-state)
+        - [Large radio button focus state](#large-radio-button-focus-state)
+        - [Large radio button selected state](#large-radio-button-selected-state)
+        - [Large radio button disabled state](#large-radio-button-disabled-state)
 - [Buttons](#buttons)
     - [Simple input with a button](#simple-input-with-a-button)
     - [Button inside an input](#button-inside-an-input)
     - [Button inside an input with a button](#button-inside-an-input-with-a-button)
 - [Select dropdown](#select-dropdown)
-    - [Default state](#default-state)
-    - [Hover/focus state](#hoverfocus-state)
-    - [Disabled state](#disabled-state)
+    - [Select dropdown default state](#select-dropdown-default-state)
```

Comment:
Same comment here as for the others. No need to repeat "Select dropdown" or "state"

Reply:
See comment above about links needing a unique name.